src/worker/vm/extensions/console.ts

```diff
@@ -1,11 +1,45 @@
 import { ConsoleExtension } from '@posthog/plugin-scaffold'
+import { captureException } from '@sentry/minimal'
+
+import { KAFKA_PLUGIN_LOG_ENTRIES } from '../../../shared/ingestion/topics'
+import { UUIDT } from '../../../shared/utils'
+import { PluginConfig, PluginLogEntry, PluginLogEntryType, PluginsServer } from '../../../types'
+
+export function createConsole(server: PluginsServer, pluginConfig: PluginConfig): ConsoleExtension {
+    function consoleRegister(type: PluginLogEntryType, ...args: unknown[]): void {
+        const entry: PluginLogEntry = {
+            id: new UUIDT().toString(),
+            team_id: pluginConfig.team_id,
+            plugin_id: pluginConfig.plugin_id,
+            timestamp: new Date().toISOString().replace('T', ' ').replace('Z', ''),
+            type,
+            message: args.map(String).join(' '),
```

Comment:
If an object passes through this, won't it come out as `[object Object]`? 

How about `JSON.stringify`? That should handle numbers, booleans, etc too - although it has its own limitations:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#description

Reply:
You're correct about objects, thanks, fixed that.